src/pages/management/configuration-api/index.mdx

```diff
@@ -3944,6 +3945,262 @@ curl -X POST \
+| Required scopes                  | `canned_responses_write`                                                          |
+| [Batch support](#batch-requests) | No                                                                            |
+
+#### Request
+
+| Parameter   | Required | Data type        | Notes                                                    |
+| ----------- | -------- | ---------------- | -------------------------------------------------------- |
+| `text`      | Yes      | `string`         | Canned response text content                         |
+| `tags`      | Yes      | `array[string]`  | Array of tags                                         |
+| `group_id`  | Yes      | `number`         | ID of the group the canned response belongs to          |
+| `is_private`| No       | `bool`           | Whether the canned response is private (default: false)  |
+
+#### Response
+
+| Parameter | Data type | Notes                        |
+| --------- | --------- | ---------------------------- |
+| `id`      | `number`  | ID of the created canned response |
+
+<Code>
+
+<CodeSample>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.7/configuration/action/create_canned_response \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "text": "Thank you for contacting us! How can I help you today?",
+    "tags": ["greeting", "support"],
+    "group_id": 1,
+    "is_private": false
+  }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+  "id": 12345
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+### Update Canned Response
+
+Updates an existing canned response. All parameters are optional - only provided parameters will be updated.
+
+#### Specifics
+
+|                                  |                                                                               |
+| -------------------------------- | ----------------------------------------------------------------------------- |
+| Method URL                       | `https://api.livechatinc.com/v3.7/configuration/action/update_canned_response` |
+| Required scopes                  | `canned_responses_write`                                                      |
+| [Batch support](#batch-requests) | No                                                                            |
+
+#### Request
+
+| Parameter   | Required | Data type        | Notes                                                    |
+| ----------- | -------- | ---------------- | -------------------------------------------------------- |
+| `id`        | Yes      | `number`         | ID of the canned response to update                      |
+| `text`      | No       | `string`         | New canned response text (max 2000 characters)          |
+| `tags`      | No       | `array[string]`  | Array of tags (max 20 tags, each max 50 characters)     |
```

Comment:
text characters and tags limits are mentioned here (update), but not for the creation; i guess it should be both or none